include/net/net_if.h

```diff
@@ -1628,6 +1628,17 @@ static inline struct in_addr *net_if_router_ipv4(struct net_if_router *router)
 struct net_if_router *net_if_ipv4_router_lookup(struct net_if *iface,
```

Comment:
The commit subject of "net/iface: Adding IPv4 router counterpart functions to IPv6" looks wrong (because IPv6 is mentioned). What about just saying "net/iface: Add IPv4 route find and add".

Reply:
ok